sopel/modules/find_updates.py

```diff
@@ -25,13 +24,9 @@
     'A new Sopel version, {}, is available. I am running {}. Please update '
     'me. Full release notes at {}'
 )
-unstable_message = (
-    'A new pre-release version, {}, is available. I am running {}. Please '
-    'update me. {}'
-)
```

Comment:
IIRC there is no use of this variable. So it was dead code.

Reply:
That's my point, though. It _shouldn't_ be unused, but it _is_.

Right now there will never be a pre-release version in `latest.json` on the website, because it's just not set up to put one there. But this code isn't really ready to handle them either, and it _should_ be.

You might not want to fix that in this PR, since it's not a "consistency" thing, but let's not just sweep out the unused variable and forget about it!